Advanced-Recruit-Training/src/BatteryStateDisplayService.cpp

```diff
@@ -24,10 +24,19 @@ BatteryStateDisplayService::~BatteryStateDisplayService()
 void BatteryStateDisplayService::handleBatteryDataReceived(const BatteryData& batteryData)
 {
     batteryStateOfChargeService_.addData(batteryData);
+    QString Time = batteryStateOfChargeService_.timeWhenChargedOrDepleted().toString("hh:mm:ss:zzz");
+    QString chargeDepletionDisplay;
+
+    if (batteryData.current < 0)
+    {
+        chargeDepletionDisplay = " Time until fully charged:";
+    }
+    else
+    {
+        chargeDepletionDisplay = " Time until fully depleted:";
+    }
 
     QTextStream(stdout) << "Voltage: " << batteryData.voltage
         << " Current: " << batteryData.current
-        << " Total Ah used: " << batteryStateOfChargeService_.totalAmpHoursUsed() << endl;
-
-    // TODO: Print out time until the battery is fully charged or depleted.
+        << " Total Ah used: " << batteryStateOfChargeService_.totalAmpHoursUsed() << chargeDepletionDisplay << Time<< endl;
```

Comment:
Need to space out `Time<<`. Also, I think we should move the `<< chargeDepletionDisplay << Time<< endl;` part to the next line, so we don't have super long one-liners.